### DISCUSSION TITLE
希望能让机器人发出来的图片是在转发里的

### DISCUSSION THREAD

**Discussion by wenqing9794** [Samarium150/mirai-console-lolicon#73] [Ideas]
![G `{NUJSTHX MD}7MB{G 2C](https://user-images.githubusercontent.com/97432165/149283947-44f796eb-d3ab-4af9-af8a-78fb36cc7b82.png)

**Comment by wenqing9794** [comment]
这样或许能降低机器人被举报或者被屏蔽的可能吧，机器人每隔一段时间群聊就被屏蔽了，老是得重启好累。大佬看看能不能增加这个功能~感谢

**Reply by Samarium150** [reply]
可以，有时间会写。不过转发好像不能降低被查水表的概率

**Reply by wenqing9794** [reply]
感谢大佬，我也是道听途说，就算不能防屏蔽，也不容易被突然的setu搞社死，也还是蛮有用的哈哈，辛苦大佬啦
